index.html

```diff
@@ -240,15 +240,29 @@ <h1>Biometrics</h1>
+    <p>Although companies and industry groups <a href="https://www.technewsworld.com/story/86512.html">boast</a> of FRT’s accuracy, FRT systems do not yet seem to be accurate enough to consistently identify individuals across various demographic groups. Even Detroit’s Police Chief has said that his department’s facial recognition software is inaccurate<a href="https://www.vice.com/amp/en_us/article/dyzykz/detroit-police-chief-facial-recognition-software-misidentifies-96-of-the-time"> 96% of the time</a>.</p>
+    <p>CPD’s Latent Print Unit (LPU), which examines fingerprints from crime scenes, has come under a great deal of <a href="https://www.bettergov.org/news/bad-police-fingerprint-work-undermines-chicago-property-crime-cases">scrutiny</a> for having opaque, ineffective, and outdated practices. The Cook County Public Defender’s Office usually <a href="https://www.documentcloud.org/documents/5765026-Henderson-Admissibility-Motions.html">motions</a> for courts to ignore testimonies from CPD’s fingerprint analysts.</p>
     <h3> <span class="glyphicon glyphicon-exclamation-sign"></span> Legal and Privacy Concerns</h3>
-    <p> The secure storage of biometric data is increasingly important as biometrics such as fingerprints are used for authentication purposes. In Illinois we have the <a href="http://www.ilga.gov/legislation/ilcs/ilcs3.asp?ActID=3004&ChapterID=57">Biometric Information Privacy Act (BIPA)</a> which regulates the collection of biometric data, by required consent as well as limits on how long the data can be stored. There are currently <a href="http://www.biometricupdate.com/201508/states-considering-biometrics-capture-laws-may-look-to-illinois-privacy-laws">lawsuits</a> against companies like Facebook and Shutterfly for collecing biometric information for commercial purposes in violation of this act. Unfortunately, law enforcement may not be regulated by BIPA. </p>
+    <p> The secure storage of biometric data is increasingly important as biometrics such as fingerprints are used for authentication purposes. In Illinois we have the <a href="http://www.ilga.gov/legislation/ilcs/ilcs3.asp?ActID=3004&ChapterID=57">Biometric Information Privacy Act (BIPA)</a> which regulates the collection of biometric data, by required consent as well as limits on how long the data can be stored. Facebook recently settled a <a href="https://www.courthousenews.com/judge-approves-650-million-settlement-in-facebook-biometric-case/">lawsuit</a> for $650 million for violating BIPA. Unfortunately, law enforcement may not be regulated by BIPA. </p>
```

Comment:
I think we should say a few things here. a) BIPA is one of the strongest biometrics privacy laws in the country. b) LPL and a bunch of our friends helped protect this law c) Law enforcement is explicitly exempt from BIPA but private companies (like Clearview) are not. d) Companies like clearview get sued for violating BIPA (see c)